packages/button/Button/DirectionalLink.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from "react"
+import arrowBackward from "@kaizen/component-library/icons/arrow-backward.icon.svg"
+import arrowForward from "@kaizen/component-library/icons/arrow-forward.icon.svg"
+import GenericButton, { GenericProps } from "./components/GenericButton"
+
+type DirectionalLinkProps = Pick<
+  GenericProps,
+  | "id"
```

Comment:
What I was thinking was that `GenericProps` should be a list of props that are common across all 4 button components, and then you can add the specific ones to each component as required (rather than having to pick the ones you want from GenericProps). I believe this is what GenericProps was designed to be, it's just that before there were probably more in common between the 2 compared to the now 4.

I can confirm that there's no external usages of GenericProps, so we wouldn't need to do a breaking change because of it either.